packages/zoe/src/contractSupport/ratio.js

```diff
@@ -74,9 +68,7 @@ export const makeRatio = (
   denominatorBrand = numeratorBrand,
 ) => {
   denominator > 0n ||
-    assert.fail(
-      X`No infinite ratios! Denominator was 0/${q(denominatorBrand)}`,
-    );
+    Fail`No infinite ratios! Denominator was 0/${q(denominatorBrand)}`;
```

Comment:
I don't like **/** as a separator here. Would you mind changing it to a space?
```suggestion
    Fail`No infinite ratios! Denominator was 0 ${q(denominatorBrand)}`;
```